public/linux.html

```diff
@@ -243,6 +243,33 @@ <h2 class="step-title">Extra: Key Modifiers</h2>
           </div>
           <div class="step-half step-diagram"></div>
         </section>
+        <section class="step" id="alternative-tools">
```

Comment:
I think this might be a candidate for a separate `<article/>`, rather than being a "step" of the existing one. You can refer to the Windows and Mac pages for an example. Doing this will hopefully lead to a better heading structure for screen readers, as well as provide a visual separation between the two methods.

Reply:
Shuffled it around to match, how's this?

Reply:
Looks great to me. Thanks!